src/main/java/com/gluonhq/substrate/target/AndroidTargetConfiguration.java

```diff
@@ -235,7 +235,7 @@ public boolean runUntilEnd() throws IOException, InterruptedException {
 
     @Override
     boolean allowHttps() {
-        return false;
+        return true;
```

Comment:
Probably better to remove the method? The default implementation already returns true.

Reply:
that is not in the JavaDoc, so we can't rely on it.
But it's better to specify the doc indeed then.